### ISSUE TITLE
Create a data product based on a DAF dataset

### ISSUE THREAD

**Issue by ruphy** [italia/daf#214]
Create an application based on a DAF dataset

You can choose any technology you want, but you should be creating an interesting dashboard, visualization, or useful program which exploits the possibilities of the DAF framework. We have several mentors available, with different expertises, so make sure you discuss with your mentor which kind of application would be useful.

Example applications:
* https://github.com/italia/daf-d3-miur-studenti-ateneo
* https://github.com/italia/daf-d3-sose-rifiuti

**Mentor**: This is a very broad task: ask on the #daf channel on https://slack.developers.italia.it

**Note**: Creativity is a big part of the deliverable, here. Try to imagine new, beautiful, intelligent ways of how you can make sense of our datasets.

**Comment by ndip007** [comment]
Hi @ruphy after going through the idea list gor GSoC, I found this project very interesting and will like to take on it. I am currently setting up the example application to try them on my own computer. 
I will probably be back soon for more highlights as far as expected deliverables are concern.
cheers

**Comment by ndip007** [comment]
@ruphy there's no readme on how to install and run the [second example](https://github.com/italia/daf-d3-sose-rifiuti) you provided. How do I install and run it?
I can't install the first since it's not adapted to run ubuntu which is the os I have

**Comment by ruphy** [comment]
Hello @ndip007,
both are simple HTML pages. They both work on Ubuntu.
The real idea which you need to come up with is interesting new uses of the datasets published. Give it a few days of thought and experimentation and try to come up with a creative proposal you can back.

**Comment by ndip007** [comment]
@ruphy , I tried following the installation guide here https://github.com/italia/daf-d3-miur-studenti-ateneo
But ran into this:
![npm_install_error](https://user-images.githubusercontent.com/26679458/36199828-06ea16fa-117b-11e8-8a1c-4d8a0a168c1e.png)

**Comment by ndip007** [comment]
@ruphy I figured out how to go about the problem. I had to update my `npm package` before running `'npm start' `command
Sorry my bad!

**Comment by urstrulykkr** [comment]
@ruphy I would like to contribute for "Implement a tool to export visualizations from notebooks and import into DAF" project. I interest in this issue.  would like to work on it :)

**Comment by urstrulykkr** [comment]
@ruphy  so we need to make some sensible outcomes on visualising the data from DAF data sets, right?
Is that what you expect ?

**Comment by ruphy** [comment]
@urstrulykkr yes, that was the idea. I am adding @jenkin to the conversation who is now working on the community of DAF. Remember you can also join the slack of Developers Italia!

**Comment by urstrulykkr** [comment]
Thanks @ruphy I had joined the concerned slack group :)
Will follow up with @jenkin

**Comment by urstrulykkr** [comment]
@jenkin  Could you please me assign to any of the DAF datasets...
